modular_skyrat/modules/customization/modules/surgery/organs/genitals.dm

```diff
@@ -184,8 +185,11 @@
 
 /obj/item/organ/genital/vagina/get_description_string(datum/sprite_accessory/genital/gas)
 	var/returned_string = "You see a [lowertext(genital_name)] vagina."
-	if(lowertext(genital_name) == "cloaca")
+	if(gas && gas.desc_string)
+		returned_string = gas.desc_string
 		returned_string = "You see a cloaca." //i deserve a pipebomb for this
+	if(genital_name == "Tauric Dog Vagina|ONLY USE ON TAUR DOGS")
```

Comment:
This if block isn't necessary anymore either because the desc_string logic up above should handle it